src/tlsuv.c

```diff
@@ -169,8 +169,10 @@ int tlsuv_stream_keepalive(tlsuv_stream_t *clt, int keepalive, unsigned int dela
 #if defined(TCP_KEEPALIVE)
         setsockopt(s, IPPROTO_TCP, TCP_KEEPALIVE, &delay, sizeof(delay));
 #endif
+#if __linux__
```

Comment:
it would be better to condition on the macro definitions -- `TCP_KEEPINTVL` and `TCP_KEEPCNT`

see above